lib/iris/experimental/regrid.py

```diff
@@ -6,8 +6,18 @@
 """
 Regridding functions.
 
-"""
+.. note::
+
+    .. deprecated:: 3.2.0
+
+    This package will be removed in a future release.
+    The PointInCell class has now moved to :class:`iris.analysis.PointInCell`.
+    All the other content will be withdrawn.
+
+    If you still use any of this, please contact the Iris Developers to
+    discuss how to replace it or to retain it.
```

Comment:
I think that this will be triggered by a regrid of a cube via `cube.regrid`, `iris.analysis.AreaWeighted` and

https://github.com/SciTools/iris/blob/9ba2adde25e6a720e6b8afd7f738ebc8663d449a/lib/iris/analysis/_area_weighted.py#L46-L48

I don't know where that leaves us?

Reply:
Sorry I hadn't thought of that.
I will fix it ...

Reply:
I have shifted the still-used code out of experimental.
Not sure I've got it right yet, though.